samples/Rich-Cards-Bot/main.py

```diff
@@ -7,245 +7,46 @@
 
 
 from aiohttp import web
-from botbuilder.schema import (Activity, ActivityTypes,
-                               AnimationCard, AudioCard, Attachment,
-                               ActionTypes, CardAction,
-                               CardImage, HeroCard,
-                               MediaUrl, ThumbnailUrl,
-                               ThumbnailCard, VideoCard,
-                               ReceiptCard, SigninCard,
-                               Fact, ReceiptItem)
+from botbuilder.schema import Activity, ActivityTypes
 from botbuilder.core import (BotFrameworkAdapter, BotFrameworkAdapterSettings, TurnContext,
                              ConversationState, MemoryStorage, UserState, CardFactory)
-"""Import AdaptiveCard content from adjacent file"""
-from adaptive_card_example import ADAPTIVE_CARD_CONTENT
+
+from dialogs import MainDialog
+from bots import RichCardsBot
 
 APP_ID = ''
 APP_PASSWORD = ''
-PORT = 9000
+PORT = 3978
```

Comment:
Do you want to add the configuration loading from `.yml` (`.yaml`?) to this sample? Or is that still mid-flight?